oak-run/src/test/java/org/apache/jackrabbit/oak/run/DataStoreCommandTest.java

```diff
@@ -322,6 +325,71 @@ public void testConsistencyVerbose() throws Exception {
         testConsistency(dump, data, true);
     }
 
+    @Test
+    public void testConsistencyVerboseRootPathNoMissing() throws Exception {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/gh/oak-indexing/jackrabbit-oak/pullRequest?prid=5026544)